core/src/main/scala/org/apache/spark/internal/io/FileCommitProtocol.scala

```diff
@@ -107,10 +111,9 @@ abstract class FileCommitProtocol extends Logging {
    * if a task is going to write out multiple files to the same dir. The file commit protocol only
    * guarantees that files written by different tasks will not conflict.
    *
-   * This API should be implemented and called, instead of
-   * [[newTaskTempFile(taskContest, dir, ext)]]. Provide a default implementation here to be
-   * backward compatible with custom [[FileCommitProtocol]] implementations before Spark 3.2.0.
+   * @since 3.2.0
    */
+  @Unstable
```

Comment:
@c21 can we mark this whole class as `@Unstable`?

Reply:
@HyukjinKwon - ah I see, updated.